src/gax/Cargo.toml

```diff
@@ -35,6 +35,7 @@ serde       = "1.0.214"
 serde_json  = "1.0.133"
 serde_with  = { version = "3.11.0", default-features = false, features = ["base64", "macros"] }
 thiserror   = "2.0.3"
+tokio       = { version = "1.42", features = ["time"] }
```

Comment:
I think this is only need for tests currently. Can we keep this out of the main deps? Maybe we are already tightly coupled to tokio with reqwest -- but if we aren't and it is not too much trouble it may be good to try to stay open to other runtimes.

Reply:
Fixed.